lambdatimes/src/components/Carousel/Carousel.js

```diff
@@ -1,35 +1,110 @@
 import React, { Component } from 'react';
-import { carouselData } from '../../data'
+import { carouselData } from '../../data';
+import styled from 'styled-components';
+
+const Carouselz = styled.div`
+width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: row;
+  height: 500px;
+  position: relative;
+  overflow: hidden;
+  margin-top: 16px;
+  img {
+    width: 100%;
+    display: none;
+  }
+  @media (min-width: 1200px) {
+      width: 1200px;
+    }
+`;
+const Buttonz= styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: none;
+  flex-direction: row;
+  color: #fff;
+  background-color: #333;
+  font-size: 40px;
+  border-radius: 50%;
+  position: absolute;
+  width: 50px;
+  height: 50px;
+  cursor: pointer;
+  top: 50%;
+  transform: translate(0, -50%);
+  &:hover{
+    color: #333;
+  background-color: #fff;
+  border: 2px solid #333;
+  }
+`;
+
+const LBut = styled(Buttonz)`
+  left: 25px;
+`
+
+const RBut = styled(Buttonz)`
+  right: 25px;
+`
+
+
+
+
 // Complete this Carousel 
 export default class Carousel extends Component {
   constructor(props){
     super(props);
     this.state = {
-
+      currentPhoto: 2,
+      photos: [],
     }
   }
   componentDidMount(){
-
+this.setState({
+  photos: carouselData,
+})
   }
 
   leftClick = () => {
-
+    if(this.state.currentPhoto > 0) {
```

Comment:
Nice job. Could have combined the logic into one function. But still good work just the same.